examples/knative/python/kn-py-vro/kn-py-vro_secret.json

```diff
@@ -0,0 +1,10 @@
+{
+    "SERVER": "VRO_FQDN_OR_IP",
+    "PORT": 443,
+    "AUTH_TYPE": "BASIC/OAUTH"
```

Comment:
nit: trailing `,` missing.

Reply:
I spotted that in my local version with real creds in but didn't fix the commited version. Will sort.